packages/orbit-components/src/NavigationBar/index.tsx

```diff
@@ -101,8 +57,17 @@ const NavigationBar = ({
   });
 
   return (
-    <StyledNavigationBar data-test={dataTest} id={id} shown={shown}>
-      <StyledNavigationBarContent>{children}</StyledNavigationBarContent>
+    <nav
+      data-test={dataTest}
+      id={id}
+      className={cx(
+        "bg-white-normal shadow-fixed p-sm z-navigation-bar fixed left-0 right-0 top-0 box-border flex w-full translate-x-0 items-center",
+        "duration-normal transition-transform ease-in-out",
```

Comment:
adding `transform-gpu` woulnd't be a bad idea, we usually prefer to use transform3d and wll-change transform to make them more smooth.